scripts/apps/search/services/SearchService.js

```diff
@@ -508,6 +508,13 @@ export function SearchService($location, config, session, multi,
         // this is needed for archived collection
         this.filter({not: {term: {package_type: 'takes'}}});
 
+        // inject custom elastic search queries
+        if (typeof params.customQueries !== 'undefined') {
```

Comment:
I would use `!= null` (with single `=`) which checks against both `null` and `undefined`

Reply:
The `typeof` operator does not return null. `typeof null === 'object'`